### PR TITLE
Disable Deterministic PDB GUID generation for RTM

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@gafter @ManishJayaswal @jaredpar

File: src/Compilers/Core/Portable/CodeAnalysis.csproj [hunk 1]
```diff
@@ -712,6 +712,7 @@
       <HintPath>..\..\..\..\packages\System.Reflection.Metadata.$(SystemReflectionMetadataVersion)\lib\portable-net45+win8\System.Reflection.Metadata.dll</HintPath>
     </Reference>
   </ItemGroup>
+<!-- Support for deterministic PDB (workitem 926)
```
Comment (jaredpar) [inline src/Compilers/Core/Portable/CodeAnalysis.csproj]: We should delete instead of commenting it out.

**Comment by jaredpar** [comment]
I'm not a fan of the `#if` approach here.  If this feature is being removed then we should remove it.

**Comment by tmat** [comment]
@jaredpar It's not being removed. It's suppressed for RTM. I'll undo this change once we snap (there is an issue open on it #926, if you are concern about #if defs left behind). A few #ifdefs are easier to merge.

**Comment by gafter** [comment]
Why are we making this change?

**Comment by jaredpar** [comment]
@tmat it's code which will forever live in our RTM code path which is never a part of our RTM deliverable.  Undoing a git commit is rather straight forward, much faster than manually undoing the edits in this change:

> https://www.atlassian.com/git/tutorials/undoing-changes/git-revert

Essentially commit the delete in the RTM branch and then `git revert` it in the appropriate branch where this change should move forward.  IMHO it's much cleaner and faster to commit a delete and then `git revert` it later on.  But this is not blocking feedback.

**Comment by tmat** [comment]
@jaredpar OK, good point.

**Comment by gafter** [comment]
@jaredpar I still don't get it. Not every line of code has to be part of a fully supported feature.

**Comment by tmat** [comment]
@jaredpar I have decided to leave the #if defs. If I deleted the parts that I effectively commented out the remaining code still contains a lot of supporting logic for deterministic PDB generation, which will have make no sense. It's not worth spending time cleaning everything up. Therefore I think it is better to keep all logic in and only comment out the parts that adds the dependencies on Diasymreader library.

**Pull request merged**